File: src/targets/node/request/fixtures/multipart-data.js
const fs = require('fs');
const request = require('request');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'content-type': 'multipart/form-data; boundary=---011000010111000001101001'},
  formData: {
    foo: {
<<<<<<< HEAD
      value: fs.createReadStream('src/fixtures/files/hello.txt'),
      options: {filename: 'src/fixtures/files/hello.txt', contentType: 'text/plain'}
=======
      value: fs.createReadStream('hello.txt'),
      options: {filename: 'hello.txt', contentType: 'text/plain'}
>>>>>>> upstream/master
    },
    bar: 'Bonjour le monde'
  }
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/python/requests/fixtures/multipart-data.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything"

files = {"foo": ("src/fixtures/files/hello.txt", open("src/fixtures/files/hello.txt", "rb"), "text/plain")}
payload = {"bar": "Bonjour le monde"}

response = requests.post(url, data=payload, files=files)

print(response.text)
=======
url = "http://mockbin.com/har"

files = { "foo": "open('hello.txt', 'rb')" }
payload = { "bar": "Bonjour le monde" }

response = requests.post(url, data=payload, files=files)

print(response.json())
>>>>>>> upstream/master
